src/Pages/Products/Components/Filter/Filters.jsx

```diff
@@ -1,64 +1,177 @@
 import React from 'react';
+import { useFilter } from '../../../../Context';
 import styles from './Filters.module.css';
 
 const Filters = () => {
+	const { state, dispatch } = useFilter();
```

Comment:
Since we are going to have multiple contexts and reducers you can probably rename ```dispatch``` to ```filterDispatch```

Reply:
Thanks, I will change that right away Kevin 🚀